### PR TITLE
Added tests for answer consensus

### PR DESCRIPTION
**Pull request by joandrsn**
This PR adds a lot of tests around the answer consensus algorithm.
Tests the following:
* That `AnswerConsensus` is created by running update_consensus
* That `AnswerConsensus` chooses the most popular vote
* That `AnswerConsensus` chooses the most popular vote after having updated consensus once
* That educations in `AnswerConsensus` is independant
* That the default value in `AnswerConsensus` is defaulted to the `Don't know`-value
* That the first question is fetched from the database.
* That if the service were behind a reverse proxy that the IP is logged.

### CODE REVIEW

**Review by Badgie — APPROVED** [review]
:taco:

File: edufinder/tests/rest_api/test_api.py [hunk 1]
```diff
@@ -308,6 +319,75 @@ def test_POST_questions_with_int_outofrange(self):
...
+    def test_answer_consensus_most_popular_add1(self):
+        self.create_answers(self.education1, AnswerChoice.NO)
+        management.call_command('update_consensus')
+        self.assertEqual(self.count_answerconsensus(self.education1, AnswerChoice.YES), Question.objects.count())
+
+    def test_answer_consensus_equal_number_add2(self):
+        self.create_answers(self.education1, AnswerChoice.NO)
+        self.create_answers(self.education1, AnswerChoice.NO)
+        management.call_command('update_consensus')
+        self.assertEqual(self.count_answerconsensus(self.education1, AnswerChoice.YES), Question.objects.count())
```
Comment (thorulf4) [inline edufinder/tests/rest_api/test_api.py:362]: Hvorfor skal denne consensus give YES? Når der er 2xNo og 2xYes

**Review by Jakobzs — COMMENTED** [review]
🕺

**Review by thorulf4 — APPROVED** [review]
Fandt ikke nødvendigvis nogen problemer er bare forvirret over en af testene

File: edufinder/tests/rest_api/test_api.py (hunk 1 above)
Comment (joandrsn) [reply edufinder/tests/rest_api/test_api.py:362]: Den giver `YES` fordi objektet bliver initialiseret med `YES` først. Når denne itereres, så starter den med `YES` og fordi der ikke er nogle hvor svarmulighedens count > `YES`'s count, så bliver det `YES`.

**Pull request merged**